### PR TITLE
Create C02_20

### PR DESCRIPTION
**Pull request by DeedeeDL**
The program will display true if an input number will be multiple of only 3 or only 7 and if it is multiple of both 3 and 7 or neither of them will display false

### CODE REVIEW

File: src/main/java/org/example/C02_20_MultipleOf.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class C02_20_MultipleOf {
```
Comment (andreiursudev) [inline src/main/java/org/example/C02_20_MultipleOf.java]: Give a better name to the class.

File: src/main/java/org/example/C02_20_MultipleOf.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your number: ");
+        int number = scanner.nextInt();
+
+        if (number % 3 == 0 ^ number % 7 == 0) {
```
Comment (andreiursudev) [inline src/main/java/org/example/C02_20_MultipleOf.java]: Dont use `^`, use only `&&` and `||`

File: src/main/java/org/example/C02_20_MultipleOf.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            System.out.println(true);
+        } else {
+            System.out.println(false);
+
```
Comment (andreiursudev) [inline src/main/java/org/example/C02_20_MultipleOf.java]: Remove empty lines.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C02_20_MultipleOfNumbers.java [hunk 4]
```diff
@@ -2,13 +2,13 @@
...
 
-public class C02_20_MultipleOf {
+public class C02_20_MultipleOfNumbers {
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         System.out.println("Enter your number: ");
         int number = scanner.nextInt();
 
-        if (number % 3 == 0 ^ number % 7 == 0) {
+        if (number % 3 == 0 || number % 7 == 0) {
```
Comment (andreiursudev) [inline src/main/java/org/example/C02_20_MultipleOfNumbers.java]: The problem says "if it is multiple of both 3 and 7 or neither of them will display false". I dont think your solution respects that.

File: src/main/java/org/example/C02_20_MultipleOfNumbers.java [hunk 5]
```diff
@@ -2,13 +2,13 @@
 
 import java.util.Scanner;
 
-public class C02_20_MultipleOf {
+public class C02_20_MultipleOfNumbers {
```
Comment (andreiursudev) [inline src/main/java/org/example/C02_20_MultipleOfNumbers.java]: The name is still to vague. Try something more specific like NumberDividedByOnly3OrOnly7.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/NumberDividedByOnly3OrOnly7.java [hunk 6]
```diff
@@ -1,7 +1,7 @@
 package org.example;
 import java.util.Scanner;
 
-public class P06_MultiplicationTable {
+public class NumberDividedByOnly3OrOnly7 {
```
Comment (andreiursudev) [inline src/main/java/org/example/NumberDividedByOnly3OrOnly7.java:4]: You renamed the wrong class.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/C02_20_MultipleOf.java (hunk 1 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C02_20_MultipleOf.java]: done

File: src/main/java/org/example/C02_20_MultipleOf.java (hunk 2 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C02_20_MultipleOf.java]: done

File: src/main/java/org/example/C02_20_MultipleOf.java (hunk 3 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C02_20_MultipleOf.java]: done

File: src/main/java/org/example/C02_20_MultipleOfNumbers.java (hunk 4 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C02_20_MultipleOfNumbers.java]: I hope now it is fine

File: src/main/java/org/example/C02_20_MultipleOfNumbers.java (hunk 5 above)
Comment (DeedeeDL) [reply src/main/java/org/example/C02_20_MultipleOfNumbers.java]: done

File: src/main/java/org/example/NumberDividedByOnly3OrOnly7.java (hunk 6 above)
Comment (DeedeeDL) [reply src/main/java/org/example/NumberDividedByOnly3OrOnly7.java:4]: done

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**